### ISSUE TITLE
Update the script to handle new affix changes in Awakening

### ISSUE THREAD

**Issue by Tvangeste** [andreberg/PoE-Item-Info#3]
Here's the post that describes the changes to the affixes in PoE Awakening Beta:

https://www.pathofexile.com/forum/view-thread/1290356

It would really nice if the script be aware of the changes.

**Comment by andreberg** [comment]
I do plan on updating for the expac. Unfortunately I have exam prep time for uni atm and it will be a while until I get to it.

**Comment by ADRDev** [comment]
I updated the data files with the info from the forum post.

Here's a zip with the files: https://drive.google.com/file/d/0ByS8vtmDxnWZdjJtRUFFbUgwQTg/view?usp=sharing

**Comment by Tvangeste** [comment]
Tony, thanks for the updated data. I've created a pull request from that archive you've posted and checked the data which seems to be correct to the best of my knowlegde.

André, will you be able to merge this pull request into the master?

**Comment by andreberg** [comment]
Yes, I will have a look at it by this weekend. Like I said in the other comments, exam tomorrow and Friday. After that I should have some time to finally give the script some attention again.

**Comment by HiPoEGH** [comment]
As I've said in the forum thread, I started working on Jewel Mods. However I stopped, I might continue or finish it in the future. Having to pull and rename all the info from poedb is tedious. 
Here is the link to what I've finished so far, hopefully you can make use of it.
https://drive.google.com/file/d/0B3ozN4g8M_L2U0IycWhERGc3X1k/view?usp=sharing

**Comment by Tvangeste** [comment]
@Hixxie thank you for you efforts, much appreciated. We badly the jewel info, I agree. Let me know when you finish the work and once previous pull requests are merged into the mainline, I will create a new request with your info for jewels. Thansk!

@andreberg I hope your exams were huge success and you have some time for this project. :)
